src/java.base/share/classes/java/util/stream/Stream.java

```diff
@@ -407,8 +407,8 @@
      *     }
      *
      *     public static void main(String[] args) {
-     *         Stream<Object> stream = ...;
-     *         Stream<Object> expandedStream = stream.mapMulti(C::expandIterable);
+     *         var nestedList = ...;
+     *         Stream<Object> expandedStream = nestedList.stream().mapMulti(C::expandIterable);
      *     }
      * }
      * }</pre>
```

Comment:
I'd suggest putting a comment somewhere outside the javadoc that points to the test and has a statement similar to the one in the test, about keeping the example here in synch with the test. I'm not sure what the best way is to do this though. It probably cannot be after the javadoc comment, because the javadoc comment needs to be immediately prior to the method declaration. Maybe a // comment above the doc comment?

Reply:
Good idea. I've added a comment above the javadoc as you've suggested. See 
944c7d5